packages/card/stories/Card.stories.tsx

```diff
@@ -7,7 +7,7 @@ import { SpaceSize } from "../../shared/styles/styleUtils/modifiers/modifierUtil
 
 const readme = require("../README.md");
 
-storiesOf("Card", module)
+storiesOf("Layout|Card", module)
```

Comment:
I'm not sure if this really belongs in the "Layout" category, but I wasn't sure where else to put it.